### PR TITLE
Fix part of #3690 : Wiki update - How to run static CI check locally

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Fix part of #3690 : Wiki Update - How to run static CI check locally

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Review by MohitGupta121 — COMMENTED** [review]
@adhiamboperes PTAL, Thanks. Please suggest if something need to change.

**Review by adhiamboperes — APPROVED** [review]

**Comment by adhiamboperes** [comment]
Hi @MohitGupta121, why are there changes in the todo line exemptions file? Did you add/remove a todo?

**Comment by MohitGupta121** [comment]
> Hi @MohitGupta121, why are there changes in the todo line exemptions file? Did you add/remove a todo?

No @adhiamboperes basically these wiki Todos are exception. So I added new line in TOC due to which these todos lines get shifted to next lines and the CI checks fail that's why I added new line number for correct TODO.

**Comment by adhiamboperes** [comment]
> > Hi @MohitGupta121, why are there changes in the todo line exemptions file? Did you add/remove a todo?
> 
> No @adhiamboperes basically these wiki Todos are exception. So I added new line in TOC due to which these todos lines get shifted to next lines and the CI checks fail that's why I added new line number for correct TODO.

Got it, thanks. This LGTM

File: wiki/Static-Analysis-Checks.md [hunk 1]
```diff
@@ -227,3 +228,11 @@ To fix the failures for this check: resolve the TODO items and then close the is
 
 # How to add a new Static analysis check
 // TODO([#3690](https://github.com/oppia/oppia-android/issues/3690)): Complete static checks Wiki
+
+# How to run static checks locally
+
+- Go to failing CI check in your GitHub PR.
```
Comment (seanlip) [inline wiki/Static-Analysis-Checks.md]: failing --> the failing

File: wiki/Static-Analysis-Checks.md [hunk 2]
```diff
@@ -227,3 +228,11 @@ To fix the failures for this check: resolve the TODO items and then close the is
...
+- Scroll failing CI check logs to the top and find the Bazel command that was run for this script.
```
Comment (seanlip) [inline wiki/Static-Analysis-Checks.md]: Scroll to the top of the failing CI check logs, and find the ...

File: wiki/Static-Analysis-Checks.md [hunk 3]
```diff
@@ -227,3 +228,11 @@ To fix the failures for this check: resolve the TODO items and then close the is
...
+- Alternatively, in android studio, go to the `.github` folder and find the [static_checks.yml](https://github.com/oppia/oppia-android/blob/develop/.github/workflows/static_checks.yml) file. Search for the line that corresponds to the name of the job that failed, and you can run the same script on your local terminal.
```
Comment (seanlip) [inline wiki/Static-Analysis-Checks.md]: android studio --> Android Studio

...job that failed. You can then run the ...

File: wiki/Static-Analysis-Checks.md [hunk 4]
```diff
@@ -227,3 +228,11 @@ To fix the failures for this check: resolve the TODO items and then close the is
...
+
+Note: Before running script command in your local terminal make sure you have Bazel installed. To know how to setup Bazel for oppia-android follow these [instructions](https://github.com/oppia/oppia-android/wiki/Oppia-Bazel-Setup-Instructions).
```
Comment (seanlip) [inline wiki/Static-Analysis-Checks.md]: Note: Before running the script command in your local terminal, make sure you have Bazel installed. To learn how to set up Bazel for Oppia Android, follow these ...

**Review by seanlip — CHANGES_REQUESTED** [review]
@MohitGupta121 Suggested some small grammar changes, PTAL. Thanks!

File: wiki/Static-Analysis-Checks.md (hunk 1 above)
Comment (MohitGupta121) [reply wiki/Static-Analysis-Checks.md]: Done.

File: wiki/Static-Analysis-Checks.md (hunk 2 above)
Comment (MohitGupta121) [reply wiki/Static-Analysis-Checks.md]: Done.

File: wiki/Static-Analysis-Checks.md (hunk 3 above)
Comment (MohitGupta121) [reply wiki/Static-Analysis-Checks.md]: Done.

File: wiki/Static-Analysis-Checks.md (hunk 4 above)
Comment (MohitGupta121) [reply wiki/Static-Analysis-Checks.md]: Done.

**Comment by MohitGupta121** [comment]
@seanlip PTAL, make the required changes. Thanks!

**Review by seanlip — APPROVED** [review]
@MohitGupta121 LGTM. Thanks!

**Pull request merged**